packages-new/components/checkbox/src/components/osds-checkbox/osds-checkbox.e2e.ts

```diff
@@ -43,24 +24,24 @@ describe('e2e:osds-checkbox', () => {
   let inputCheckedProperty: boolean;
   let activeElementId: string | undefined;
   const logger = new OdsLogger('e2e:osds-checkbox');
-  const spyEvent = async (eventName: keyof OdsCheckboxEvents) => await el.spyOnEvent(eventName);
+  const spyEvent = async (eventName: keyof OdsCheckboxEvent) => await el.spyOnEvent(eventName);
 
   async function setup({
                          attributes = {},
                          nativeAttributes = {},
                          html = '',
                          htmlOutside = '',
                          onPage,
-                       }: { attributes?: Partial<OdsCheckboxAttributes>, html?: string, htmlOutside?: string, nativeAttributes?: Partial<HTMLElement>, onPage?: ({ page }: { page: E2EPage }) => void } = {}) {
-    const minimalAttributes: OdsCheckboxAttributes = OdsCreateAttributes(attributes, odsCheckboxBaseAttributes);
-    const stringAttributes = OdsComponentAttributes2StringAttributes<OdsCheckboxAttributes>(minimalAttributes, odsCheckboxDefaultAttributes);
-    const nativeStringAttributes = OdsComponentAttributes2StringAttributes<Partial<HTMLElement>>(nativeAttributes, {});
+                       }: { attributes?: Partial<OdsCheckboxAttribute>, html?: string, htmlOutside?: string, nativeAttributes?: Partial<HTMLElement>, onPage?: ({ page }: { page: E2EPage }) => void } = {}) {
+    const baseAttribute = { ariaLabel: '', checked: false, disabled: false, hasFocus: false, updating: false, value: '' };
```

Comment:
as the value never change, you can declare it on top, otherwise a new var is created on each call